File: backend/spec/factories/ingredient_nutrients.rb
# frozen_string_literal: true

FactoryBot.define do
  factory :ingredient_nutrient do
    ingredient
    nutrient
    content_quantity { 1 }
    content_unit { 'kcal' }
    content_unit_per { 100 }
    content_unit_per_unit { 'g' }
  end
end

File: graphqlapi/src/graphql/queries/ingredient_nutrients.ts
import { gql } from "@apollo/client";

export const INGREDIENT_NUTRIENTS_QUERY = gql`
  query GetIngredientNutrients($limit: Int) {
    ingredientNutrients(limit: $limit) {
      id
      ingredient {
        name
      }
      nutrient {
        name
      }
      contentQuantity
      contentUnit
      contentUnitPer
      contentUnitPerUnit
    }
  }
`;



